source/sass/blocks/photo.scss

```
@@ -45,6 +45,10 @@
   }
 }
 
+img .photo__img--top {
```

Как что-то может лежать в картинке???
```suggestion
.photo__img--top {
```